tests/commonlib/utils.py

```diff
@@ -304,6 +305,30 @@ def get_findings(elastic_client, config_timeout, query, sort, match_type):
     return result
 
 
+def get_telemetry(config):
+    """
+    This function create eAPI call to Kibana snapshot telemetry api and return is payload.
+    Example:
+    @param config: configuration object contains kibana host and auth
+    @return: Telemetry payload
+    """
+    method = "POST"
+    url = f"{config.kibana_url}/internal/telemetry/clusters/_stats"
+    headers = {
+        "Content-Type": "application/json",
+        "kbn-xsrf": "true",
+        "elastic-api-version": "2",
+        "x-elastic-internal-origin": "Kibana",
+    }
+    auth = config.basic_auth
+
+    response = requests.request(method=method, url=url, headers=headers, auth=auth, json={"unencrypted": "true"})
```

Comment:
@gurevichdmitry implemented a wrapper we use around the codebase. I would recommend using it for consistency if possible. https://github.com/elastic/cloudbeat/blob/82d2cb7b3912b2c36f8bd88f90fd933829b904fa/deploy/test-environments/fleet_api/src/api/base_call_api.py#L36

Reply:
@gurevichdmitry and I discussed on that point, to reuse `perform_api_call` in the sanity tests module we should convert the fleet_api module to python package.

Reply:
@oren-zohar, as @CohenIdo mentioned, to use the `perform_api_call`, we need to build the `fleet_api` package for reuse. Since the request is made only once, it's better to implement it the way Ido did. However, I agree that building the package is one of the tasks that can be added for refactoring.